src/editor/mainEditor/material/logic/view/BasicMaterialView.ts

```diff
@@ -3,3 +3,8 @@ import { setColor as setColorBuss } from "../bussiness/BasicMaterialBuss";
 export const setColor = (colorStr:string) => {
     setColorBuss(colorStr);
 };
+
+export const getColor = () => {
+    // I guess I uneed this from bottom layer
+    return "#FFFFFF"
```

Comment:
I need an interface to get color from bottom layer.

Reply:
ok, i will add it